src/components/PopupWithForm.js

```diff
@@ -18,17 +19,51 @@ export default class PopupWithForm extends Popup {
     return formValues;
   }
 
+  _checkFormValidity() {
+    const isFormValid = Array.from(this._inputList).every(
+      (input) => input.value.trim() !== ""
+    );
+    this._submitButton.disabled = !isFormValid;
+    if (isFormValid) {
+      this._submitButton.classList.remove("button_inactive");
+    } else {
+      this._submitButton.classList.add("button_inactive");
```

Comment:
Please, delete `_checkFormValidity` from `PopupWithForm`

Now only class `FormValidator` should **`disable/enable`** submit buttons and remove errors from the inputs (if you do it). It’s a violation of **Single responsibility** principle